consensus/src/config.rs

```diff
@@ -32,3 +32,5 @@ pub const RELAX_ITERATION_THRESHOLD: u8 = 10;
 
 pub const ROUND_BASE_TIMEOUT: Duration = Duration::from_secs(5);
 pub const MAX_STEP_TIMEOUT: Duration = Duration::from_secs(60);
+
+pub const MAX_BLOCK_SECS_DRIFT: u64 = 120;
```

Comment:
I would strongly suggest to make this much larger.
The whole point of this PR is to remove the timestamp from the block logic validity, since it should have no significance in it.

We are keeping this check as a sanity check to avoid absurd timestamps. A 120-second time shift is definitely not absurd and still very risky IMHO.